columnphysics/icepack_atmo.F90

```diff
@@ -271,6 +271,8 @@ psixh     ! stability function at zlvl   (heat and water)
 
       k = 1
       do while (abs(ustar - ustar_prev)/ustar > atmiter_conv .and. k <= natmiter)
+         k = k + 1
+         ustar_prev = ustar
```

Comment:
Looks like a bug fix! Thanks for catching this. Why are these two lines not at the end of the loop though?

Reply:
`ustar_prev = ustar` can't be at the end of the loop, because it would not be ustar_prev if it was.  It doesn't matter if `k=k+1` is at the start or end of the loop, but I have changed this so that it is consistent with the ocean coupling routine while loop now in CIME.

Reply:
You are correct! Looks good. Approving.